### ISSUE TITLE
Restrict Host Console Access to SACONFIG

### ISSUE THREAD

**Issue by Wild1145** [TotalFreedom/TotalFreedomMod#2124]
Currently it appears the console can do anything it likes with the saconfig command, I would suggest we restrict this to just updating admin details (Re-Adding basically). Thoughts?

**Comment by ncurran02** [comment]
@Wild1145, Console is already restricted to telnet permission, as set in the configuration.

**Comment by Wild1145** [comment]
@Focusvity But I could still add random people to it without being able to be traced back. Im of the opinion that host consoles should not need to do anything more than add imposters to admin, as you can get on telnet the rest of the time?

**Comment by ncurran02** [comment]
You do make a valid point.

**Comment by JeromSar** [comment]
I agree, but the level of console permissions should be configurable. Currently, we already have an option to set whether the host console is considered senior admin or telnet admin.
